src/config.rs

```diff
@@ -0,0 +1,1235 @@
+use std::{collections::HashSet, path::PathBuf, time::Duration};
+
+use anyhow::{Error, Result};
+use quick_xml::{events::Event, Reader};
+use serde::Deserialize;
+
+const EXPECTED_DOCTYPE_PARTS: &[&str] = &[
+    "busconfig",
+    "PUBLIC",
+    r#""-//freedesktop//DTD"#,
+    "D-Bus",
+    "Bus",
+    "Configuration",
+    r#"1.0//EN""#,
+    r#""http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd""#,
+];
+
+const STANDARD_SYSTEM_SERVICEDIRS: &[&str] = &[
+    "/usr/local/share/dbus-1/system-services",
+    "/usr/share/dbus-1/system-services",
+    "/lib/dbus-1/system-services",
+];
+
```

Comment:
Nit: I've been trying to keep the code in top-down in zbus and would like to do the same in busd: pub items before private and use before definition. I see that you've mostly followed that pattern but it also means this private const should be moved below all the public API.